projects/cfengine/project.yaml

```diff
@@ -4,6 +4,10 @@ language: c++
 primary_contact: "vratislav.podzimek@northern.tech"
 auto_ccs:
   - "Adam@adalogics.com"
+fuzzing_engines:
```

Comment:
Is it OK to skip these?
I thought we will need to support Centipede eventually...

Reply:
Another question:
I noticed that you disabled some projects, [even though they build successfully in my PR](https://docs.google.com/spreadsheets/d/1Hs0vzS0wZdW-6DYHF_BxJUIrPuKi1Oy0tD--3mAIBcQ/edit#gid=2090256933) (I highlighted them in the sheet, `537db38f` is my build ID and `9b99ff13` is yours.)
I also tested `curl` on your branch locally, and found it can build and pass the `bad_build_test`.
Would you mind reminding me why they are disabled? Thanks!

Reply:
I think it's probably going to be a long time before projects won't be able to use oss-fuzz unless they support centipede.